router/batchrouter/asyncdestinationmanager/bing-ads/util.go

```diff
@@ -278,17 +286,17 @@ In the below format (only adding relevant keys)
 		{"1<<>>client2", "error2", "Customer List Item Error"},
 	}
 */
-func ReadPollResults(filePath string) [][]string {
+func (b *BingAdsBulkUploader) ReadPollResults(filePath string) [][]string {
 	// Open the CSV file
 	file, err := os.Open(filePath)
 	if err != nil {
-		log.Fatal("Error opening the CSV file:", err)
+		b.logger.Errorf("Error opening the CSV file: %w", err)
```

Comment:
don't we need to propagate error to the caller?

Reply:
done